refactor(data-toolbar): move and wrap chips (#2397)
patternfly/patternfly-next: src/patternfly/components/DataToolbar/examples/DataToolbar.md

| -- | -- | -- |
| `.pf-m-show{-on-[breakpoint]}` | `.pf-c-data-toolbar__group.pf-m-toggle-group`, `.pf-c-data-toolbar__expandable-content` | Modifies toolbar element visibility at breakpoint. This selector must be applied consistently to toggle group and expandable content. |
| `.pf-m-toggle-group-container` | `.pf-c-data-toolbar__content` | Modifies the data toolbar content for toggle group, expandable content layout. |

> | `.pf-m-chip-container` | `.pf-c-data-toolbar__group` | Modifies the data toolbar group for selected filters applied filters layout. |

Reviewer: ```suggestion
| `.pf-m-chip-container` | `.pf-c-data-toolbar__expandable-content` | Modifies the data toolbar expandable content for selected filters applied filters layout. |
```

            Status
          {{/select}}
        {{/data-toolbar-item}}
        {{#> data-toolbar-item}}
          {{#> select id=(concat data-toolbar--id '-select-checkbox-risk-expanded') select--IsCheckboxSelect="true" select--ItemIsSelected="true"}}
            Risk
          {{/select}}
        {{/data-toolbar-item}}
        {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-chip-container"}}

>           {{> data-toolbar-item-chip-group chip-group--label="Status" chip-group--id=(concat data-toolbar--id '-group1-')}}

Reviewer: do you need the trailing `-`?

```suggestion
          {{> data-toolbar-item-chip-group chip-group--label="Status" chip-group--id=(concat data-toolbar--id '-group1')}}
```

Author: the trailing dash constructs a complex `id` correctly

>           {{> data-toolbar-item-chip-group chip-group--label="Risk" chip-group--id=(concat data-toolbar--id '-group2-')}}

Reviewer: same here

```suggestion
          {{> data-toolbar-item-chip-group chip-group--label="Risk" chip-group--id=(concat data-toolbar--id '-group2')}}
```

Reviewer: looks like this is in a few other spots, too, I won't tag 'em all unless you want me to

Author: same here, the trailing dash constructs a complex `id` correctly
